waf_dl_unsupported.go

```diff
@@ -15,8 +15,10 @@ import (
 
 type wafDl struct{}
 
+var UnsupportedTargetErr = &UnsupportedTargetError{fmt.Errorf("the target operating-system %s or architecture %s are not supported", runtime.GOOS, runtime.GOARCH)}
```

Comment:
I don't think we need this variable to be public

Reply:
Fixed by 28f73a9